src/printer/stats.rs

```diff
@@ -0,0 +1,122 @@
+use std::{
+    ops::{Add, AddAssign},
+    time::Duration,
+};
+
+use super::util::NiceDuration;
+
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
+#[cfg_attr(feature = "serde1", derive(serde::Serialize))]
```

Comment:
Oh I should strip this `serde1` feature attribute, I was attempting to remove its existence (ie make it always compile with the `serde` support)

Reply:
Does the "1" there just mean "version 1 of serde"?

Reply:
I'm not sure what the "1" means, it looks like that feature controls whether the `serde*` dependencies are included/utilized

Specifically it looks like the JSON printer depends on it